whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -156,6 +159,69 @@ public Object mapSearch(final String requestUrl, final List<RpslObject> objects,
         return mapCommons(searchResult, requestUrl);
     }
 
+    public Object mapOrganisationEntity(final String requestUrl, List<RpslObject> objects,
```

Comment:
`List<RpslObject>` can be 100,000's in size. We should refactor `RdapQueryHandler.handleQuery()` to accept a `ResponseHandler` or return an `Iterable` of RpslObjects instead.